src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -1596,6 +1597,16 @@ public Territory getCurrentBattleTerritory() {
   public IBattle getCurrentBattle() {
     return m_currentBattle;
   }
+
+  protected void raids() {
```

Comment:
Some quick low level comments. Method should be moved up to minimize vertical distance between usage and declaration. Please see the examples at: http://www.triplea-game.org/dev_docs/dev/code_standards, "Method and Variable ordering"

Also looks like this method can be made private.